### ISSUE TITLE
The majority of the examples don't run or show nothing

### ISSUE THREAD

**Issue by holgerbrandl** [scenerygraphics/scenery#73]
The landing page [suggests](https://github.com/scenerygraphics/scenery#examples) to run the examples to dive into scenery. However the majority of the examples listed under  [src/test/tests/graphics/scenery/tests/examples](https://github.com/scenerygraphics/scenery/blob/master/src/test/tests/graphics/scenery/tests/examples) don't run, lack resources or show nothing.

Imho the cave specific stuff is not so well suited as a beginner example, because most users are unlikely to own a cave. Some of the demos should have more documentation to indicate what is actually 

- [x] ArcballTargetExample

Black window without any object

- [x] BileExample

Does not work. Relies on local resources like `M:/meshes/adult_mouse_bile_canaliculi_network_2.stl`

```
Unable to load native library: /private/var/folders/cl/j57y4y2504n1nnlyd4q41fj40000gp/T/vrpn-natives-tmp1381526963179001382/libjava_vrpn.dylib: dlopen(/private/var/folders/cl/j57y4y2504n1nnlyd4q41fj40000gp/T/vrpn-natives-tmp1381526963179001382/libjava_vrpn.dylib, 1): Library not loaded: /usr/local/opt/libusb/lib/libusb-1.0.0.dylib
  Referenced from: /private/var/folders/cl/j57y4y2504n1nnlyd4q41fj40000gp/T/vrpn-natives-tmp1381526963179001382/libjava_vrpn.dylib
  Reason: image not found
Did you include jvrpn-natives-mac in your dependencies?
[main] INFO VRPNTrackerInput - Initializing VRPN device DTrack at 10.1.2.201
/private/var/folders/cl/j57y4y2504n1nnlyd4q41fj40000gp/T/vrpn-natives-tmp1381526963179001382/libjava_vrpn.dylib: dlopen(/private/var/folders/cl/j57y4y2504n1nnlyd4q41fj40000gp/T/vrpn-natives-tmp1381526963179001382/libjava_vrpn.dylib, 1): Library not loaded: /usr/local/opt/libusb/lib/libusb-1.0.0.dylib
  Referenced from: /private/var/folders/cl/j57y4y2504n1nnlyd4q41fj40000gp/T/vrpn-natives-tmp1381526963179001382/libjava_vrpn.dylib
  Reason: image not found
Error initializing java_vrpn.
 -- Unable to find native library.
Error initializing remote device DTrack@10.1.2.201.
 -- Unable to find the right functions.  This may be a version problem.

java.lang.InstantiationException: vrpn.TrackerRemote.init(Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)Z

	at vrpn.VRPNDevice.<init>(VRPNDevice.java:61)
	at vrpn.TrackerRemote.<init>(TrackerRemote.java:80)
	at graphics.scenery.controls.VRPNTrackerInput.initializeTracker(VRPNTrackerInput.kt:62)
	at graphics.scenery.controls.VRPNTrackerInput.setTrackerAddress(VRPNTrackerInput.kt:32)
	at graphics.scenery.controls.VRPNTrackerInput.<init>(VRPNTrackerInput.kt:49)
	at graphics.scenery.controls.TrackedStereoGlasses.<init>(TrackedStereoGlasses.kt:24)
	at graphics.scenery.tests.examples.BileExample.init(BileExample.kt:33)
	at graphics.scenery.SceneryDefaultApplication.main(SceneryDefaultApplication.kt:123)
	at graphics.scenery.tests.examples.BileExample.main(BileExample.kt:99)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:51)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)

```

- [x] BloodCellsExample

works just great

- [x] BoxedProteinExample

odd output without visible protein, maybe because `ORC6.obj` is not part of demo data?

- [x] ClusterExample

does not work. same error as bile

- [x] CubeExample

works great

- [x] DemoReelExample

does not work. same error as bile

- [x] FontRenderingExample

works great

- [x] LineExample

works great

- [x] MultiBoxExample

just gray frame is shown

- [x] MultiBoxInstancedExample

just light source without any box

- [x] OpenVRExample

just black scene without objects
_Edit: Has been removed_

- [x] OrientationBoxesExample

does not work. same error as bile

- [x] PowerplantExample

Does not work because  `powerplant.obj` is not part of `scenery-demo-models`

- [x] RungholtExample

Does not work because  `rungholt.obj` is not part of `scenery-demo-models`

- [x] SponzaExample

does not work. same error as bile.

Also used path to `sponza.obj` is not the same as in `scenery-demo-models`
_Edit: Fixed path._

- [x] TexturedArrowExample

Does look more like a Galgen, but fine otherwise
_Edit: Has been removed_

- [x] TexturedCubeExample

works great

- [x] TexturedCubeJavaExample

works great

- [x] TexturedCubeJavaJPanelExample

throws NPE
```
kotlin.KotlinNullPointerException
	at graphics.scenery.SceneryDefaultJPanelApplication.main(SceneryDefaultJPanelApplication.kt:82)
	at graphics.scenery.tests.examples.TexturedCubeJavaJPanelExample.testExample(TexturedCubeJavaJPanelExample.java:17)

```
_Edit: Has been removed_

- [x] VertexUpdateExample

just black frame

- [x] VolumeBoxExample

does not work. same error as bile
_Edit: Will be removed_

- [x] VolumeDrosoExample

does not work. same error as bile
_Edit: Will be removed_

- [x] VolumeExample

does not work. same error as bile
_Edit: Updated in #74, example data will be added_

- [x] VolumeHistoneExample

does not work. same error as bile
_Edit: Will be removed_

- [x] VRControllerExample

gray scene with light source. It's unclear what the demo is actually about.

My system is a 2016 macbook pro with "Radeon Pro 460 4096 MB + Intel HD Graphics 530 1536 MB"

Edit by @skalarproduktraum: Converted to task list

**Comment by skalarproduktraum** [comment]
Hi @holgerbrandl!

Thanks for the extensive report! 
I'm aware that things are broken at the moment on master, but the README on master also states

```
Hello, this is the master branch, where development happens and stuff breaks. If you require a stable version of scenery, please check out the latest stable version of this repository.
```

But as said, I need to go through the examples anyway, clean then up or remove them, so I'll use this issue to keep track of that.

**Comment by skalarproduktraum** [comment]
@holgerbrandl If you have time, have a look at the fix-examples branch or #75 -- all the basic and advanced examples should be back to working.
